core/solidus_core.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'activemerchant', '~> 1.66'
   s.add_dependency 'acts_as_list', '~> 0.3'
   s.add_dependency 'awesome_nested_set', '~> 3.0', '>= 3.0.1'
-  s.add_dependency 'cancancan', '~> 2.2'
+  s.add_dependency 'cancancan', '~> 3.0'
```

Comment:
It seams we are still able to support cancancan 2.x as well? We should not force everybody else to upgrade to 3.0 as well. Could we set a lower bound of `>= 2.2` and upper bound of `< 4.0`?